gridss.wdl

```diff
@@ -33,6 +33,7 @@ task GRIDSS {
         BwaIndex reference
         String outputPrefix = "gridss"
 
+        String jvmheapsize = "25G"
```

Comment:
You could make this an Int, so it can be used in the runtime section as to set the memory requirement as well.

And then call it something like `jvmHeapSizeGb`